hoot-core/src/main/cpp/hoot/core/io/PbfReader.cpp

```diff
@@ -153,14 +153,62 @@ void PbfReader::_addTag(shared_ptr<Element> e, QString key, QString value)
     }
     else
     {
-      e->setCircularError(_circularError);
-      if (_badAccuracyCount < 10)
+      bool isBad = false;
+      Tags t1;
+      t1.setList(key, QStringList() << value);
+      //check if value contian velocity units
+      if (value.contains("km/h", Qt::CaseInsensitive) || value.contains("kph", Qt::CaseInsensitive)
+          || value.contains("kmph", Qt::CaseInsensitive) || value.contains("mph", Qt::CaseInsensitive)
+          || value.contains("knots", Qt::CaseInsensitive))
```

Comment:
The error:circular and corresponding accuracy tags should only ever store length values, not velocities.